docs/commands_states.rst

```diff
@@ -179,6 +192,288 @@ Notes and caveats
 
 .. note:: Would a command-line interface be useful?
 
+Commands archive v2
+-------------------
+
+Version 2 of the commands archive is currently provided as an experimental
+release to allow getting experience with the new interface.
+
+For details of the commands v2 archive, including important information about
+network access, the timeliness of commands, configuration and process details,
+please see:
+
+.. toctree::
+   :maxdepth: 2
+
+   commands_v2.rst
+
+
+Getting commands
+^^^^^^^^^^^^^^^^
+
+In order to use the v2 version do the following::
+
+  >>> from kadi import commands
+  >>> commands.conf.commands_archive_version = "2"  # must be the string "2" not int 2
```

Comment:
Is this "commands_archive_version" or "commands_version"?

Reply:
Fixed.